common/src/java/org/apache/hadoop/hive/ql/ErrorMsg.java

```diff
@@ -471,6 +471,7 @@
           "Not an ordered-set aggregate function: {0}. WITHIN GROUP clause is not allowed.", true),
   WITHIN_GROUP_PARAMETER_MISMATCH(10422,
           "The number of hypothetical direct arguments ({0}) must match the number of ordering columns ({1})", true),
+  AMBIGUOUS_STRUCT_FIELD(10423, "Struct field is not unique: {0}", true),
```

Comment:
nit: Usually we use "field" for row types and "attribute" for struct types. Plus it reads more natural if we inline the attribute name in the sentence: 

`Attribute \"{0}\" specified more than once in structured type.`

Reply:
Renamed `field` to `attribute`